### ISSUE TITLE
Assets is returning empty array

### ISSUE THREAD

**Issue by ilyich-erd** [multiversx/mx-sdk-js-network-providers#15]
I'm calling 
```js 
apiNetworkProvider.getNonFungibleToken(
      collection,
      nonce
);
```

in it's response `assets` is an empty array `[]`.

I think this issue is related to:
https://github.com/ElrondNetwork/elrond-sdk-erdjs-network-providers/blob/af289fa7e89107d26c9b6a3827579906063cf046/src/tokens.ts#L80

since `payload` does not have `assets` key, it instead has `uris`.

Is it something that we can fix?

**Comment by andreibancioiu** [comment]
Hello @ilyichvismara, and thank you for the details :pray: 

We will most probably include a fix for this in https://github.com/ElrondNetwork/elrond-sdk-erdjs-network-providers/pull/14.

However, until then, you can switch to performing direct HTTP requests against the API (and using your own, tailored data structures). This is our current recommendation:

> The default classes `ApiNetworkProvider` / `ProxyNetworkProvider` should **only be used as a starting point**. As your dApp matures, make sure you **switch to using your own network provider**, tailored to your requirements (whether deriving from the default ones or writing a new one, from scratch) that directly interacts with the Elrond API (or Gateway).

Further reference:
 - https://docs.elrond.com/sdk-and-tools/erdjs/extending-erdjs/

**Comment by ilyichvismara** [comment]
> Hello @ilyichvismara, and thank you for the details 🙏
> 
> We will most probably include a fix for this in #14.
> 
> However, until then, you can switch to performing direct HTTP requests against the API (and using your own, tailored data structures). This is our current recommendation:
> 
> > The default classes `ApiNetworkProvider` / `ProxyNetworkProvider` should **only be used as a starting point**. As your dApp matures, make sure you **switch to using your own network provider**, tailored to your requirements (whether deriving from the default ones or writing a new one, from scratch) that directly interacts with the Elrond API (or Gateway).
> 
> Further reference:
> 
> * https://docs.elrond.com/sdk-and-tools/erdjs/extending-erdjs/

Yeah It's what I did, thanks for the incoming fix
